src/screens/Customers/components/TariffPlanInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { createStyles, withStyles, Theme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { CardPropsInfo } from '../../../components/OutlinedCardList';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      marginTop: '10px',
+      border: '1px solid',
+      borderColor: theme.palette.divider,
+      borderRadius: '6px',
+    },
+    content: {
+      flex: 1,
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      padding: '8px',
+    },
+    item: {
+      width: '100%',
+      color: theme.palette.text.primary,
+    },
+    label: {
+      whiteSpace: 'pre-line',
+      lineHeight: '1.2',
+      marginTop: '5px',
+    },
+    gray: {
+      color: 'rgba(189,189,189,1)',
+    },
+  });
+
+interface TariffPlanInfoProps {
+  classes: { root: string; content: string; item: string; gray: string; label: string; };
+  undefinedAmount: string;
+  meterType: string;
+  fixedTariff: string;
+  unitTariff: string;
+  minUnits: string;
+}
+
+function TariffPlanInfo(props: TariffPlanInfoProps) {
+  const { classes, undefinedAmount, meterType, fixedTariff, unitTariff, minUnits } = props;
+
+  if (meterType === "Inactive") {
```

Comment:
Also, I don't know if we need a `meterType === 'Inactive'` because that would just mean the customer is inactive right? We can swap this out later, but just wanted to know if there's a distinction that I don't know about

Reply:
oooh true ill add a Todo there